lib/interface/cli/commands/pipeline/run.base.js

```diff
@@ -14,7 +14,9 @@ class RunBaseCommand {
 
     async run() {
         await this.preRunRequest();
-        const { debug, skip, only, trigger, cpu, memory, branch, sha, disk } = this.argv;
+        const {
+            debug, skip, only, trigger, cpu, memory, branch, sha, disk, shadowRunning,
+        } = this.argv;
```

Comment:
why do we need the `shadowRunning` flag.
in case not passing pipeline name backend and cli should validated that the yaml file descriptor was passed, if not will throw an error